proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -360,6 +363,11 @@ message ConsumerValidator {
   int64 power = 2;
   // public key the validator uses on the consumer chain during this epoch
   tendermint.crypto.PublicKey consumer_public_key = 3;
+  // height the validator had when it FIRST became a consumer validator
+  // If a validator becomes a consumer validator at height `H` and is continuously a consumer validator for all the upcoming
+  // epochs, then the height of the validator SHOULD remain `H`. This height only resets to a different height if a validator
+  // stops being a consumer validator during an epoch and later becomes again a consumer validator.
+  int64 height = 4;
 }
```

Comment:
nit: rename to something more intuitive, e.g., init_height, join_height...

Reply:
Good point. `JoinHeight` is a much better name than simply `Height`. I renamed it.